packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -145,7 +145,7 @@ function createJob(job, { reporter }) {
   }
 
   promise.catch(err => {
-    reporter.panic(err)
+    reporter.panic(`error converting image`, err)
```

Comment:
I missed the original PR, but the capitalisation on this could do with fixing. Obviously not needed in the cherrypick.

Reply:
True, but I didn't want to NIT pick about that given that currently error is just hidden because we show some joi validation error instead.

I will open PR with capitalization, but those will not need any backporting because it's nicety ;)

Reply:
See for before/after - https://github.com/gatsbyjs/gatsby/pull/28291#pullrequestreview-540011488

We actually don't seem to show that error message at all (so lack of capitalization is least of problems to figure out). But at least we do show what was thrown.

I didn't exactly reproduce same problem that user had (not sure how), but I just simulated it by throwing object with `name` field

Reply:
I already have a ch ticket to catch image worker errors and log something useful, so I'll look at that when I come to it